lib/agent/providers/files/index.js

```diff
@@ -151,6 +151,11 @@ module.exports.get_tree = function(options, cb) {
       return cb(new Error('Unable to parse files data'));
     }
 
+    if (files.length == 0) {
+      logger.info("files"+files.length);
```

Comment:
It would be better if the logger info alert something like "0 files in folder" or something like that.